lib/front_matter.ts

```diff
@@ -93,12 +105,12 @@ function escapeYAML(str: string) {
 }
 
 interface Options {
-  mode?: 'json' | '',
-  prefixSeparator?: boolean,
-  separator?: string
+	mode?: 'json' | '';
+	prefixSeparator?: boolean;
+	separator?: string;
 }
```

Comment:
Prefers the previous delimiter.

Reply:
oh yes sorry i havent lint fix it, next commit i ran `eslint --fix` to fix them.

Reply:
fixed https://github.com/dimaslanjaka/hexo-front-matter/commit/5d35fe8507f8727db849f1cde60a0b893e716eb8

Reply:
refactor: implement lint-staged and husky

to automatically fix code style before commited

-    https://github.com/dimaslanjaka/hexo-front-matter/commit/6ec68d7d737dd12f3441b7093af745e0b416677c
-    https://github.com/dimaslanjaka/hexo-front-matter/commit/87dd66e6ce413194f04b7c4c1bd7b9a204a7ddbf